recipes-aos/aos-iamanager/files/encrypt.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+echo "hello" | cryptsetup  -q --type luks2 luksFormat /dev/hda6
+
+PASSWORD=hello  PIN=1234 systemd-cryptenroll --pkcs11-token-uri="pkcs11:token=aos;object=diskencryption" /dev/hda6
+
+echo "hello" | cryptsetup luksRemoveKey /dev/hda6
+
+PIN=1234 /lib/systemd/systemd-cryptsetup attach aos_partition /dev/hda6 - pkcs11-uri=auto
```

Comment:
Looks like violation of policies.

Reply:
this implementation is only for a virtual machines for testing porpuses